benefits/flexible_holiday.md

```diff
@@ -1,12 +1,12 @@
-# Untracked Holiday
+# Flexible Holiday
 
 We're building a high-performance organisation that delivers great software products. We recognise that this requires a lot of hard work and that for our team to perform at their highest possible level, they need to live a well-balanced life and take regular time-out for rest, relaxation and rejuvenation.
 
-We have adopted an untracked holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. 
+We have adopted a flexible holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. We ensure everyone in the team take at least the minimum 23 days as per our employment contract though welcome everyone to take more than this.
```

Comment:
Nitpicking: the contract specifies this as an entitlement rather than a minimum. I think I'd just state the 23 days minimum in here without referencing the contract